web/services/AI/ai_model.py

```
@@ -0,0 +1,180 @@
+* **Tone:** If you want to specify the tone of the feedback (e.g., encouraging, critical, neutral), include it in the prompt.
+    Example: "Provide feedback in an encouraging and supportive tone."
+* **Error Handling:** Add a instruction to handle cases where the submitted assignment is not related to the expected subject,
+    or is completely unreadable. Example: "if the submitted assignment is unrelated to the expected subject, or is unreadable,
+    return a degree of 0, and explain the reason in the student and teacher feedback."
+
+
+**Example of a more complete prompt:**
+'''
+
+
+def ai_assignment_corrector(example_form):
```

Ruff (0.8.2)

Missing return type annotation for public function `ai_assignment_corrector`

(ANN201)